src/LibHdb++Cassandra.cpp

```diff
@@ -81,10 +83,14 @@ HdbPPCassandra::HdbPPCassandra(vector<string> configuration)
     {
         local_dc = LOCAL_DC_DEFAULT;
 
-        LOG(Debug) << "Library configuration parameter local_dc is not defined. Defaulting to: " 
+        LOG(Info) << "Library configuration parameter local_dc is not defined. Defaulting to: " 
                    << LOCAL_DC_DEFAULT << endl;
     }
 
+    // ---- store diag times optional config parameter ----
+    auto store_diag_time_str = get_config_param(libhdb_conf, "store_diag_time", false);
+    store_diag_time_str == "true" ? _store_diag_times = true : _store_diag_times = false;
```

Comment:
We should document the new configuration parameters in the doc.

Reply:
They are documented in the header with the rest. Not sure we document them much elsewhere. Currently need to update and add new info at readthedocs, so lets document them there.